tests/unit/protocol-notification-bar.js

```diff
@@ -0,0 +1,42 @@
+/* global describe, beforeEach, it, spyOn */
+
+describe('protocol-notification.js', function() {
+
+    'use strict';
+
+    beforeEach(function () {
+        var details =
+            '<button class="mzp-c-button">Open Notification</button>';
+
+        document.documentElement.insertAdjacentHTML('afterbegin', details);
+    });
+
+    describe('interactions', function () {
+
+        var options = {
+            open: function () {}, // eslint-disable-line no-empty-function
+            close: function () {} // eslint-disable-line no-empty-function
+        };
+
+        spyOn(options, 'open');
+        spyOn(options, 'close');
+
+        var button = document.querySelector('.mzp-c-button');
+
+        button.addEventListener('click', function (e) {
+            e.preventDefault();
+
+            Mzp.Notification.init( {
```

Comment:
The test is failing because the first parameter of the function should be the origin (`button`), and the second parameter should be the `options` (assumign you remove the redundant `content` parameter in the above comment).

I'd also suggest moving this code inside the `it()` function below. If there are to be multiple tests, and this is code that should be run before each, then it should go inside a `beforeEach()` function.